corehq/apps/app_manager/tests/test_suite.py

```diff
@@ -1033,6 +1033,48 @@ def _load_case(self, child_module_form, case_type, parent_module=None):
             module.parent_select.module_id = parent_module.unique_id
 
 
+class UserCaseOnlyModuleAsChildTest(BasicModuleAsChildTest):
+    """
+    Even though a module might be usercase-only, if it acts as a parent module
+    then the user should still be prompted for a case of the parent module's
+    case type.
```

Comment:
If you create amodule and a form that doesn't modify any cases, but either preloads or updates the usercase, then it's considered a "usercase-only module". You can set this module as a parent module.